test/system-test/conftest.py

```diff
@@ -145,11 +145,21 @@ def setup_ccf_session():
 
 
 def _setup_kms():
-    if USE_AKV:
-        call_script([
-            "./scripts/akv/key-import.sh",
-            f'{os.getenv("DEPLOYMENT_NAME", "kms")}-private-key'
-        ])
+    if USE_AKV and TEST_ENVIRONMENT == "ccf/sandbox_local":
```

Comment:
How does this relate to the check vs 127.0.0.1:8000? I guess this is a different scenario with a real AKV?